### DISCUSSION TITLE
blocked page on the Adguard Parental control web service

### DISCUSSION THREAD

**Discussion by mgadda406** [AdguardTeam/AdGuardHome#5307] [General]
Hello All

How can I change the look of the page that appears when pages are blocked on the Adguard Parental control web service?

Thanks.

**Comment by ammnt** [comment]
> Hello All
> 
> How can I change the look of the page that appears when pages are blocked on the Adguard Parental control web service?
> 
> Thanks.

You can change the block pages addresses via AdGuardHome.yaml config file:

```
dns:
  parental_block_host: 0.0.0.0
  safebrowsing_block_host: 0.0.0.0
```

More information here:
https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#customblock
